packages/desktop-client/src/components/schedules/ScheduleDetails.jsx

```diff
@@ -216,6 +222,18 @@ export function ScheduleDetails({ modalProps, actions, id }) {
           _conditions: [{ op: 'isapprox', field: 'date', value: date }],
           _actions: [],
         };
+        if (fromTrans) {
+          schedule._account = transaction.account;
+          schedule._amount = transaction.amount;
+          schedule._amountOp = 'is';
+          schedule._date = {
+            frequency: 'monthly',
+            start: transaction.date,
+            patterns: [],
+          };
+          schedule.name = payees[transaction.payee].name;
```

Comment:
<!-- Warning represent user-facing problems. If possible, it's great to follow this kind of comment with a suggestion.  -->
**🔨 warning:** ‏what if the transaction doesn't have a payee?

Reply:
Done